### DISCUSSION TITLE
Does anyone have any API version of this app hosted externally?

### DISCUSSION THREAD

**Discussion by Frooodle** [robbrad/UKBinCollectionData#446] [General]
I am wanting to get a arduino hooked up to the output of this application but cant host a server 24/7 to maintain its own output etc
Is there a web API anyone is hosting out there to use?

**Comment by dp247** [comment]
I don't think so, but it's an idea for the future.

**Reply by robbrad** [reply]
Unless we fund infrastructure it's going to be difficult to do this.

**Reply by dp247** [reply]
Agreed. I'd rather not pay myself, only if sponsorships pay will it be viable lol

**Reply by Frooodle** [reply]
My thought is more for the code of this to be available for this so that others can host it themselves not for a global instance

**Reply by Frooodle** [reply]
I'd love to put it on a spare VPS and share it with friends etc (for reference I have 3d printed some bin indicator light things and setup some Arduino code)

**Reply by robbrad** [reply]
I'll make a docker container for this and an api. Shouldn't take long to nock something up

**Reply by Frooodle** [reply]
Works perfectly! thanks!
